### PR TITLE
Fix multiple kubeconfig bug

### PR DESCRIPTION
**Pull request by yolossn**

### CODE REVIEW

File: backend/cmd/headlamp.go [hunk 1]
```diff
@@ -209,14 +210,27 @@ func createHeadlampHandler(config *HeadlampConfig) http.Handler {
...
-		var err error
-
-		contextsFound, err := GetContextsFromKubeConfigFile(kubeConfigPath)
-		if err != nil {
-			log.Println("Failed to get contexts from", kubeConfigPath, err)
+		var kubeConfigs []string
+		if runtime.GOOS == "windows" {
+			kubeConfigs = strings.Split(kubeConfigPath, ";")
+		} else {
+			kubeConfigs = strings.Split(kubeConfigPath, ":")
```
Comment (joaquimrocha) [inline backend/cmd/headlamp.go:217]: Nit: I'd assign the right delimiter to a var and avoid duplicating the logic (i.e. calling just one Split).

File: backend/cmd/headlamp.go [hunk 2]
```diff
@@ -209,14 +210,27 @@ func createHeadlampHandler(config *HeadlampConfig) http.Handler {
...
+		for _, kubeConfig := range kubeConfigs {
+			kubeConfig, err := absPath(kubeConfig)
+			if err != nil {
+				log.Println("Failed to get current user", err)
+				continue
+			}
+
+			contextsFound, err := GetContextsFromKubeConfigFile(kubeConfig)
+			if err != nil {
+				log.Println("Failed to get contexts from", kubeConfig, err)
```
Comment (joaquimrocha) [inline backend/cmd/headlamp.go:229]: Should we print these errors to stderr?

**Review by joaquimrocha — APPROVED** [review]
Left a couple of comments. One is a nit, the other (stderr) may require a bit more closer check.
Approving nonetheless since it looks good.

Would be great if we could have tests for this too, but if that's complicated for now, we can merge and do that later.

File: backend/cmd/headlamp.go (hunk 2 above)
Comment (yolossn) [reply backend/cmd/headlamp.go:229]: All the logs are in Println, Ill fix it as a part of another PR ?

**Comment by yolossn** [comment]
> Would be great if we could have tests for this too, but if that's complicated for now, we can merge and do that later.

Will require a refactor and to get this tested, lets do it later ?

**Pull request merged**